google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerServiceContributor.java

```diff
@@ -53,26 +51,21 @@ public void contribute(StandardServiceRegistryBuilder serviceRegistryBuilder) {
         SpannerDialect.class.getName())) {
       serviceRegistryBuilder
           // The custom Hibernate schema management tool for Spanner.
-          .addInitiator(new StandardServiceInitiator() {
-            @Override
-            public Service initiateService(Map configurationValues,
-                ServiceRegistryImplementor registry) {
-              return SCHEMA_MANAGEMENT_TOOL;
-            }
-  
-            @Override
-            public Class getServiceInitiated() {
-              return SchemaManagementTool.class;
-            }
-          })
+          .addInitiator(
+              new StandardServiceInitiator() {
+                @Override
+                public Service initiateService(
+                    Map configurationValues, ServiceRegistryImplementor registry) {
+                  return SCHEMA_MANAGEMENT_TOOL;
+                }
+
+                @Override
+                public Class getServiceInitiated() {
+                  return SchemaManagementTool.class;
+                }
+              })
           // The user agent JDBC connection property to identify the library.
-          .applySetting("hibernate.connection.userAgent", HIBERNATE_API_CLIENT_LIB_TOKEN)
-          // Create a unique index for a table if it does not already exist when in UPDATE mode.
-          .applySetting(
```

Comment:
1. The `InlineIdsOrClauseBulkIdStrategy` class has been removed in Hibernate 6.
2. The `UniqueConstraintSchemaUpdateStrategy.RECREATE_QUIETLY` causes Hibernate 6 to create the same unique index multiple times, because it uses random names.

Both the above should be tested once we have a fully working Hibernate 6 dialect (I'll make a note about this).